### DISCUSSION TITLE
Invoice template field project description

### DISCUSSION THREAD

**Discussion by chrisonline** [kimai/kimai#3742] [Questions & Answers] [answered]
How can I print the project description inside the invoice?
The default one has `entry.project.name` so I have tried `entry.project.description`, but without success.

I am talking bout this field inside the "project":
![image](https://user-images.githubusercontent.com/2077218/211213413-26c19e37-385c-45be-b5da-022ac3aea1a3.png)

**Comment by kevinpapst** [comment, marked as answer]
https://www.kimai.org/documentation/invoices.html#project-variables

**Reply by chrisonline** [reply]
Sorry :-)
I had read this page multiple times, but didn't see the field.

**Reply by kevinpapst** [reply]
That's because you expected `description` ... but it was called `comment` for the first two years and then it was to late to change it 😁
